### ISSUE TITLE
[FR] Pandoc Compatible Syntax (fenced-divs)

### ISSUE THREAD

**Issue by Servinjesus1** [tnichols217/obsidian-columns#52]
Multicolumn Markdown is available in Pandoc via a [columns](https://github.com/dialoa/columns) filter. While this isn't native pandoc, the filter seems prolific and compatible enough to be considered part of the pandoc ecosystem, and the default go-to for pandoc multicolumns. It follows the [`fenced_div`](https://pandoc.org/MANUAL.html#divs-and-spans) syntax of native Pandoc, which makes sense as the multi-column environment is a type of div.

I would love to see this plugin adopt this syntax so that my markdown documents are compatible with pandoc. Right now, the syntax available is extensive, which I appreciate for mitigating any sense of [vendor lock-in](https://en.wikipedia.org/wiki/Vendor_lock-in), however I'm not sure if any of the existing syntaxes derive from any multi-platform tools or existing Markdown standards. A pandoc-compatible syntax would let Obsidian (and its markdown syntax) remain interoperable with other formats via pandoc.

**Comment by tnichols217** [comment]
Yes, i actually considered this syntax when i originally built this plugin, however since i do not do the markdown processing myself and instead pull it from the obsidian API, there's no way to give it different tokens to look for (like the ::: token) theoretically it can be done, but without support for live preview though so i might add that as a feature in the future
